mscore/pagesettings.h

```diff
@@ -22,29 +22,42 @@
 #define __PAGESETTINGS_H__
 
 #include "ui_pagesettings.h"
+#include "libmscore/mscore.h"
+#include "libmscore/page.h"
 
 namespace Ms {
 
 class MasterScore;
 class Score;
 class Navigator;
 
+// For converting from Ms::Units to a two-char string used as a suffix
+static const char* unitSuffixes[] = {"px", "mm", "in", "sp"};
```

Comment:
OK.  As for milliseconds, decibels, bpm, etc - those are not at all related to PageFormat.  Do you really need to include those in this PR?  That's a bunch more files to touch.  It seems that the % and empty string might be outside that scope as well, since they appear to be unique to drumroll.cpp and pianoroll.cpp (even though "roll" means something entirely different in those file names).